cypress/integration/error-handling.spec.js

```diff
@@ -69,7 +69,7 @@ describe( 'Test error handling of the Query Building', () => {
 		cy.get( '.query-condition:nth(1) .delete-condition-button' ).click();
 
 		// Run query
-		cy.get( '.querybuilder__run .wikit-Button' ).click();
+		cy.get( '.querybuilder__run .wikit-Button--progressive' ).click();
```

Comment:
we should make this a variable. so we don't have to change names everywhere.
something like this at the top:

`const runQueryButton = cy.get('.querybuilder__run .wikit-Button--progressive')
`

and then use this variable everywhere needed.

`runQueryButton.click();`

Reply:
Done

Reply:
It errors with:
```error: Do not assign the return value of a Cypress command (cypress/no-assigning-return-values) at cypress/integration/error-handling.spec.js:14:3:
```
I make the text a variable.